templates/zerver/landing_nav.html

```diff
@@ -7,174 +7,252 @@
+                <a href="https://github.com/zulip/zulip" target="_blank" rel="noopener noreferrer">GitHub</a>
+            </div>
+        </details>
+        <details>
+            <summary class="top-menu-mobile__item-summary">Solutions</summary>
+            <div class="top-menu-mobile__submenu">
+                <span class="top-menu-mobile__submenu-section">USE CASES</span>
+                <a href="/for/business/">Business</a>
+                <a href="/for/education/">Education</a>
+                <a href="/for/research/">Research</a>
+                <a href="/for/events/">Events and conferences</a>
+                <a href="/for/open-source/">Open source projects</a>
+                <a href="/for/communities/">Communities</a>
+                <span class="top-menu-mobile__submenu-section">CUSTOMER STORIES</span>
+                <a href="/case-studies/idrift/">Efficient distributed team management at iDrift AS</a>
+                <a href="/case-studies/tum/">Organized chat for 1000s of students at TUM</a>
+                <a href="/case-studies/ucsd/">Communication hub across 6 continents at UCSD</a>
+                <a href="/case-studies/lean/">Research collaboration at scale in the Lean mathematical community</a>
+                <a href="/case-studies/asciidoctor/">Inclusive discussion in the open-source Asciidoctor community</a>
+                <a href="/case-studies/rust/">Faster decision-making in the Rust language community</a>
+                <a href="/case-studies/recurse-center/">Platform for a worldwide community since 2013 at Recurse Center</a>
+                <a href="/communities/">Open communities directory</a>
+            </div>
+        </details>
+        <details>
+            <summary class="top-menu-mobile__item-summary">Resources</summary>
+            <div class="top-menu-mobile__submenu">
+                <span class="top-menu-mobile__submenu-section">FOR USERS</span>
+                <a href="/help/getting-started-with-zulip">Getting started</a>
+                <a href="/help">Help center</a>
+                <a href="https://zulip.com/development-community/" target="_blank" rel="noopener noreferrer">Community chat</a>
+                <a href="/help/contact-support">Contact support</a>
+                <span class="top-menu-mobile__submenu-section">FOR ADMINISTRATORS</span>
+                <a href="/help/getting-your-organization-started-with-zulip">Setting up your organization</a>
+                <a href="https://zulip.readthedocs.io/en/stable/production/install.html">Installing a Zulip server</a>
+                <a href="https://zulip.readthedocs.io/en/stable/production/upgrade-or-modify.html">Upgrading a Zulip server</a>
+                <a href="https://github.com/zulip/zulip" target="_blank" rel="noopener noreferrer">GitHub</a>
+            </div>
+        </details>
+        <a href="https://zulip.com/plans/">Pricing</a>
+        <a href="/apps/">Download</a>
+    </div>
```

Comment:
Is this the exact same content as in the non-mobile view? If so, it'd be really nice if we could avoid duplicating it by just having the difference in visuals for mobile sizes be implemented using CSS media queries rather than having two copies of everything.

(I suspect that's the kind of thing that's annoying to do in the design tool, but ideally we would do in an implementation, because otherwise we have to constantly try to avoid accidental drift between the two copies).

Reply:
Event though the overall content is same, the structure in which it is displayed is very different. So, I don't see a way to  merge them here.